app/assets/javascripts/redux/modules/models/listenSignals.js

```diff
@@ -142,6 +146,25 @@ export const reducer = handleActions({
 
   [LISTEN_SIGNALS_PUT_REQUEST_FAIL]: handlesListenSignalFailResponse,
 
+  [LISTEN_SIGNALS_DELETE_REQUEST]: (state, action) => ({
+    ...state,
+  }),
+
+  [LISTEN_SIGNALS_DELETE_REQUEST_SUCCESS]: (state, action) => {
+    return {
+      ...state,
+      data: {
+        ..._.omit(state.data, action.meta.signal.id),
+      },
+    };
+  },
+
+  [LISTEN_SIGNALS_DELETE_REQUEST_FAIL]: (state, action) => ({
```

Comment:
This error should go on the particular listen signal, not on the whole listen signal store.